src/test/java/com/epam/tc/hw3/AbstractChromeTest.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.tc.hw3;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import utils.Config;
+
+public class AbstractChromeTest {
+
+    protected WebDriver driver;
+    static WebDriverWait webDriverWait;
+    public static List<String> leftMenuItems =  List
+            .of("Home", "Contact form", "Service", "Metals & Colors", "Elements packs");
+    public static List<String> LIST_ITEMS_HEADER = List
+            .of("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS");
```

Comment:
Why do you declare leftMenuItems and LIST_ITEMS_HEADER variables in base class? You only need it in Exercise1Test class. Also, it's better to use uppercase.